go.mod

```diff
@@ -23,6 +22,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/5GC-DEV/config5g-cdac v0.2.1 // indirect
```

Comment:
@gab-arrobo - this change may not be good right? We should be using omec-project config5g. We can not be using forked version of config5g.